content/docs/rfcs/10/README.md

```diff
@@ -209,6 +209,48 @@ The accounting model can be used in `13/WAKU2-STORE` and `12/WAKU2-FILTER` to pr
 Additionally, this gives node operators who provide a useful service to the network an incentive to perform that service.
 See [18/WAKU2-SWAP](/spec/18) for more details on this piece of work.
 
+## Appendix A: Implementation Notes
+
+### Implementation Matrix
+
+There are multiple implementations of Waku v2 and its protocols:
+
+- [nim-waku (Nim)](https://github.com/status-im/nim-waku/)
+- [nim-go (Go)](https://github.com/status-im/go-waku/)
+- [nim-js (NodeJS and Browser)](https://github.com/status-im/js-waku/)
```

Comment:
```suggestion
- [go-waku (Go)](https://github.com/status-im/go-waku/)
- [js-waku (NodeJS and Browser)](https://github.com/status-im/js-waku/)
```
I often do the same mistake :)

Reply:
sigh I thought I fixed this one :p thanks